maps/tether/submaps/tether_misc.dmm

```diff
@@ -2671,6 +2671,12 @@
 	initial_flooring = null
 	},
 /area/space)
+"Fv" = (
+/turf/unsimulated/mineral{
+	icon = 'icons/turf/transit_vr.dmi';
+	icon_state = "rock"
+	},
+/area/holodeck/source_smoleworld)

```

Comment:
Looks like you have one stray tile of holo-smoleworld above the in-transit tram tracks, and that's breaking the holodeck.

Reply:
WHoops... I had a feeling I did a miss click somewhere because after testing it asked me to save. Good eye.

Reply:
Map should be fixed now.